engine/jobmaster/dm/api_test.go

```diff
@@ -378,7 +395,20 @@ func TestQueryStatusAPI(t *testing.T) {
 		]
 	}
 }`
+	for taskID, status := range jobStatus.TaskStatus {
+		require.Equal(t, status.Duration/time.Minute, duration/time.Minute)
+		status.Duration = 0
+		jobStatus.TaskStatus[taskID] = status
+	}
+	for taskID, finishedStatus := range jobStatus.FinishedUnitStatus {
+		expected := finishedTaskState.FinishedUnitStatus[taskID]
+		for i, status := range finishedStatus {
+			require.Equal(t, status.Duration, expected[i].Duration)
+			status.Duration = 0
+		}
+	}
```

Comment:
here tests real data @lance6716

Reply:
we can't get the precise time for `expectedStatus`. So I just set `Duration = 0` to pass the L412 more conveniently, and check it L398-409

Reply:
you can set it to, for example, `status.Duration = timee.Minute`, and then the JSON output is

```
"Duration": 60000000000,
```

then reviewer will notice that the unit should be included in the name to reduce ambiguity, like `"duration_in_nanosec": 60000000000,`